ssz/src/main/java/net/consensys/cava/ssz/SSZWriter.java

```diff
@@ -240,9 +241,9 @@ default void writeHash(Bytes hash) {
   }
 
   /**
-   * Write a list of bytes.
+   * Write a (varags) list of bytes.
    *
-   * @param elements The bytes to write as a list.
+   * @param elements The bytes to write as a varags list.
```

Comment:
I don't think this is necessary (also, it's not a "varargs list" that is returned).

Reply:
Cool, I'm fine either way. It's a shame that list is overloaded in Java.